dsl/demo/Gemfile

```diff
@@ -0,0 +1,2 @@
+gem "roast-ai", path: "../.."
+gem "plugin-gem-example", path: "../plugin-gem-example"
```

Comment:
This ensures our demo is always using the local versions of gems so our examples are valid.